commit c6de6ed3ac609445221e896063ad41856d4b47b8 (refs/changes/64/664/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-12T22:32:59-07:00 (3 years ago)
    
    RELENG-612 git+ssh package.json entries should be git+https (git+ is required to differentiate git repo from tarball)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 75997d7..d2bb190 100644
--- a/package.json
+++ b/package.json
@@ -12,15 +12,15 @@
     "assert-plus": "^1.0.0",
     "bunyan": "^1.8.0",
     "jsprim": "^1.2.2",
-    "krill": "https://github.com/misterdjules/node-krill.git#fix-empty-predicate-ldap-conversion",
+    "krill": "git+https://github.com/misterdjules/node-krill.git#fix-empty-predicate-ldap-conversion",
     "ldap-filter": "^0.3.1",
     "ldapjs": "^1.0.0",
     "libuuid": "^0.1.4",
     "mkdirp": "^0.5.1",
-    "moray": "https://github.com/joyent/node-moray.git#b84ef0e",
+    "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
     "once": "^1.3.3",
     "restify": "^4.0.4",
-    "sdc-clients": "https://github.com/joyent/node-sdc-clients.git#2743b5265679ca0e17d3770cf671c3e3b7d577cd",
+    "sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#2743b5265679ca0e17d3770cf671c3e3b7d577cd",
     "trace-event": "^1.3.0",
     "vasync": "^1.6.4"
   },
